scripts/jest/visual-adapter.js

```diff
@@ -34,11 +45,13 @@ const BoxVisualTestUtils = {
     },
 
     // Takes image screenshots after user input, e.g., clicking or entering text
-    takeScreenshotAfterInput: async (id, selector, action = 'click', userInput) => {
+    takeScreenshotAfterInput: async (id, selector, action = 'click', userInput, afterInputSelector = null) => {
         await BoxVisualTestUtils.gotoStory(id);
-        await global.page.waitForSelector(selector);
+        await global.page.waitForSelector(selector, { visible: true });
         await global.page[action](selector, userInput);
-        await global.page.waitFor(100);
+        (await afterInputSelector) !== null
+            ? await global.page.waitForSelector(afterInputSelector, { visible: true })
+            : await global.page.waitFor(125);
```

Comment:
i think it was a diff of like 3 test suites, if i went higher it would have been a diff of 6 ish test suites.  I wasnt looking at how many tests because they were ranging from 1-3 test within them